sktime/forecasting/callbacks/callback_list.py

```diff
@@ -0,0 +1,52 @@
+"""Callback list calls all callbacks in the list."""
+from sktime.forecasting.callbacks.callback import Callback
+
+
+class CallbackList(Callback):
+    """Callback list calls all callbacks in the list."""
+
+    def __init__(self, callbacks=None, forecaster=None, scores=None):
+        if callbacks is None:
+            callbacks = []
```

Comment:
Can you not directly self.callbacks =[] ?

Reply:
PyCharm IDE is complaining that I am passing a mutable object as a default. Not sure in which corner cases that could be a problem, i also thought the same as you but then because of the warning i left it as None. Any ideas?

Reply:
I guess you meant changing the default value, does not make sense in this line to set `self.callbacks`.

Reply:
Got it now, sorry. lgtm